src/orquestra/sdk/_base/cli/_dorq/_repos.py

```diff
@@ -107,15 +110,42 @@ class ConfigRepo:
     Wraps accessing ~/.orquestra/config.json
     """
 
-    def list_config_names(self) -> t.Sequence[ConfigName]:
-        config_entries = sdk.RuntimeConfig.list_configs()
-        return [
-            # .list_configs() should probably already return "ray" and "in_process".
-            # TODO: fix that in https://zapatacomputing.atlassian.net/browse/ORQSDK-674
-            _config.RAY_CONFIG_NAME_ALIAS,
-            _config.IN_PROCESS_CONFIG_NAME,
-            *config_entries,
-        ]
+    @staticmethod
+    def list_config_names() -> t.Sequence[ConfigName]:
+        return sdk.RuntimeConfig.list_configs()
+
+    @staticmethod
+    def store_token_in_config(uri, token):
+        runtime_name = RuntimeName.QE_REMOTE
+        config_name = _config.generate_config_name(runtime_name, uri)
+
+        config = sdk.RuntimeConfig(
+            runtime_name,
+            name=config_name,
+            bypass_factory_methods=True,
+        )
+        setattr(config, "uri", uri)
+        setattr(config, "token", token)
+        _config.save_or_update(config_name, runtime_name, config._get_runtime_options())
+
+        return config_name
+
+
+class QeClientRepo:
+    """
+    Wraps access to QE client
+    """
+
+    @staticmethod
+    def get_login_url(uri: str):
+        client = _client.QEClient(session=requests.Session(), base_uri=uri)
+        # Ask QE for the login url to log in to the platform
+        try:
+            target_url = client.get_login_url()
+        except (requests.ConnectionError, requests.exceptions.MissingSchema):
+            print(f"Unable to communicate with server: {uri}", file=sys.stderr)
+            raise exceptions.UnauthorizedError(f'Cannot connect to server "{uri}"')
```

Comment:
**issue**: this mixes the data layer and presentation layer, we shouldn't print anything here. Please handle the exception printing in a presenter object.

Reply:
See: https://github.com/zapatacomputing/orquestra-workflow-sdk/blob/eb40be5f4b0df11fd722ded0890c5b66903e1d69/src/orquestra/sdk/_base/cli/_dorq/_ui/_presenters.py#L57-L60

Reply:
Yeh this print doesnt bring any value. its leftover from development which I forgot to remove